Gemfile

```diff
@@ -5,10 +5,21 @@ source 'https://rubygems.org'
 # Specify your gem's dependencies in spree-spl.gemspec
 gemspec
 
+gem 'byebug'
```

Comment:
nie powinniśmy wrzucić tego w `group :development, :test` ? No bo raczej ogólnie to nam nie jest potrzebne.
Więc ja bym to przeniosła